File: frontend/src/app/modules/customer/models/customer.model.ts
import {TasklionAccountModel} from "../../tasklion-account/models/tasklion-account.model";
import {PersonalDetailModel} from "../../tasklion-account/models/personal-detail.model";

export interface CustomerModel {

  tasklionAccount: TasklionAccountModel;
  personalDetail: PersonalDetailModel;
}

File: backend/src/main/java/com/tasklion/backend/features/tasklionUser/customer/CustomerModel.java
package com.tasklion.backend.features.tasklionUser.customer;

import com.tasklion.backend.features.personalDetail.PersonalDetailModel;
import com.tasklion.backend.features.tasklionAccount.model.TasklionAccountModel;
import com.tasklion.backend.features.tasklionUser.TasklionUserModel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@EqualsAndHashCode(callSuper = true)
@Data
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
public class CustomerModel extends TasklionUserModel {

    private TasklionAccountModel tasklionAccount;
    private PersonalDetailModel personalDetail;

}
